ovirt_imageio/client/_ovirt.py

```diff
@@ -47,6 +47,7 @@ def connect(args):
         username=args.username,
         password=args.password,
         ca_file=args.cafile,
+        insecure =not args.secure,
```

Comment:
Should be:

    insecure=not args.secure,

You can detect such errors by running tox locally.

Reply:
thank you for the review, uploading now.